brambling/views/organizer.py

```diff
@@ -1009,5 +1009,26 @@ def get_context_data(self, **kwargs):
         context.update({
             'event': self.event,
             'event_admin_nav': get_event_admin_nav(self.event, self.request),
+            'table': FinanceTable(self.event, context['transactions']),
         })
         return context
+
+    def render_to_response(self, context, *args, **kwargs):
+        "Return a response in the requested format."
+
+        format_ = self.request.GET.get('format', default='html')
+
+        if format_ == 'csv':
+            context = super(FinancesView, self).get_context_data(**kwargs)
+            table = FinanceTable(self.event, context['transactions'])
+
+            pseudo_buffer = Echo()
+            writer = csv.writer(pseudo_buffer)
+            response = StreamingHttpResponse((writer.writerow([unicode(cell.value) for cell in row])
```

Comment:
I think this should maybe stay unicode(cell), since that has some extra internal handling for iterables? Though we don't (and probably won't) need that for this table. But we do use it for ModelTables, and I'd like to minimize the differences between the APIs. (Alternately, convince me that we should move in this direction for ModelTables?)